pkg/events/definition.go

```diff
@@ -5,9 +5,18 @@ import (
 )
 
 type Definition struct {
-	id           ID // TODO: use id ?
-	id32Bit      ID
-	name         string
+	id      ID // TODO: use id ?
+	id32Bit ID
+	name    string
+
+	// 1. Major field is bumped whenever some data the event used to have was changed
+	// e.g. a field was renamed or removed
+	// 2. Minor field is bumped whenever a non breaking change occurs
+	// e.g. a new field was added to the event
+	// 3. Patch field is bumped whenever something is changed in the way the event works internally
+	// e.g. some bug was fixed in the code
+	version      string
```

Comment:
Maybe we should set the version to be a struct so we can enforce the use of semantic versioning:
```
type version struct {
    int major
    int minor
    int patch
}
```

WDYT?

Reply:
@yanivagman at first I thought a string is simpler, and we could validate at the event definition creation, but it doesn't block someone from creating it without the constructor and have a bad version, so if going with a struct, maybe let's use the one from the semver package direclty? https://github.com/Masterminds/semver/blob/master/version.go#L47

Reply:
AH, I remembered why I chose not to validate now, because the signatures are using their own versioning schema https://github.com/aquasecurity/tracee/blob/main/signatures/golang/docker_abuse.go#L27, enforcing semver now will require research to change their signatues, is it something we want to do now?